bcda/auth/middleware.go

```diff
@@ -64,7 +64,12 @@ func RequireTokenAuth(next http.Handler) http.Handler {
 				ctx := context.WithValue(r.Context(), "token", token)
 				next.ServeHTTP(w, r.WithContext(ctx))
 				return
+			} else {
+				log.Error("Blacklisted Token with ID: %v was rejected", token.Claims.(jwt.MapClaims)["id"])
 			}
+
+		} else {
+			log.Error("Invalid Token with ID: %v was rejected", token.Claims.(jwt.MapClaims)["id"])
```

Comment:
I know enough to be concerned that this is where PII could get logged, but not enough to be certain the preceding logic is enough to have already filtered it out.  It should be a well-formed token, but still . . .  What's the worst case we could see get logged here?

Reply:
Right now only tokens signed with our key pair will work. So this should only ever be the UUID of the token thast we put in it when we created it.  I am ONLY logging when we find the expected claims to prevent logging when someone presents "bearer <SOCIAL_SECURITY_NUMBER>" as their token.